packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -982,7 +982,7 @@ class ClipRectLayer extends ContainerLayer {
   }
 
   /// {@template flutter.clipper.clipBehavior}
-  /// Controls how to clip (default to [Clip.antiAlias]).
+  /// Controls how to clip (default to [Clip.hardEdge]).
```

Comment:
`defaults to` sounds slightly better? Not sure what the grammar overlords will say, but it sounds better to me.